code/modules/admin/topic/topic_events.dm

```diff
@@ -140,13 +148,30 @@
 			if(!H.hud_used)
```

Comment:
You should update the single var usage here (T and H) to use a more descriptive name.

See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#use-descriptive-and-obvious-names